<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dashboard</class>
 <widget class="QMainWindow" name="dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2083</width>
    <height>1482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dashboard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="imageDownloadButton">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>430</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: rgb(182, 182, 182, 74); border-radius: 25px}
background-image: url(:/resources/icon-ios7-refresh-128.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>140</y>
      <width>351</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{border-radius: 25px; background-color: rgb(255, 255, 255, 0);};
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="zipCodeEdit">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>430</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>98119</string>
    </property>
   </widget>
   <widget class="QLabel" name="pictureFrameLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>140</y>
      <width>331</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{border-radius: 25px; background-color: rgb(95, 90, 131);};
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pictureLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>160</y>
      <width>271</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>                              Picture</string>
    </property>
   </widget>
   <widget class="QLabel" name="weatherIconLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(182, 182, 182, 75);</string>
    </property>
    <property name="text">
     <string>weather</string>
    </property>
   </widget>
   <widget class="QLabel" name="weatherTempLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>221</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>80</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>67</string>
    </property>
   </widget>
   <widget class="QLabel" name="weatherDescLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>desc</string>
    </property>
   </widget>
   <widget class="QLabel" name="weatherHiLoLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>14</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>hi/lo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="weatherDownloadButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>260</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: rgb(255, 255, 255, 0); border-radius: 25px}
background-image: url(:/resources/icon-ios7-refresh-128.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hourLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(255, 255, 255);};
</string>
    </property>
    <property name="text">
     <string>22</string>
    </property>
   </widget>
   <widget class="QLabel" name="minuteLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>22</string>
    </property>
   </widget>
   <widget class="QLabel" name="secondLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>22</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeColon">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QLabel" name="backgroundLabel">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-20</y>
      <width>1371</width>
      <height>721</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="degreeLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>o</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentDateLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Tue, Oct. 9</string>
    </property>
   </widget>
   <widget class="QLabel" name="russiaHourLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>350</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(255, 255, 255);};
</string>
    </property>
    <property name="text">
     <string>22</string>
    </property>
   </widget>
   <widget class="QLabel" name="russiaTimeColon">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>350</y>
      <width>21</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QLabel" name="russiaMinuteLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>22</string>
    </property>
   </widget>
   <widget class="QLabel" name="israelHourLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>440</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(255, 255, 255);};
</string>
    </property>
    <property name="text">
     <string>22</string>
    </property>
   </widget>
   <widget class="QLabel" name="israelTimeColon">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>440</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QLabel" name="israelMinuteLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>440</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>22</string>
    </property>
   </widget>
   <widget class="QLabel" name="russiaTimeLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Condensed</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Russia</string>
    </property>
   </widget>
   <widget class="QLabel" name="israelTimeLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>410</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Condensed</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Israel</string>
    </property>
   </widget>
   <widget class="QLabel" name="newYorkHourLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>530</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(255, 255, 255);};
</string>
    </property>
    <property name="text">
     <string>22</string>
    </property>
   </widget>
   <widget class="QLabel" name="newYorkTimeColon">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>530</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QLabel" name="newYorkMinuteLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>530</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>22</string>
    </property>
   </widget>
   <widget class="QLabel" name="newYorkTimeLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>500</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Condensed</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>New York</string>
    </property>
   </widget>
   <widget class="QLabel" name="greetingsLabel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>90</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Good morning, Logan</string>
    </property>
   </widget>
   <widget class="QLabel" name="zipLabel">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>410</y>
      <width>57</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Condensed</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ZIP</string>
    </property>
   </widget>
   <zorder>backgroundLabel</zorder>
   <zorder>imageLabel</zorder>
   <zorder>imageDownloadButton</zorder>
   <zorder>zipCodeEdit</zorder>
   <zorder>pictureFrameLabel</zorder>
   <zorder>pictureLabel</zorder>
   <zorder>weatherTempLabel</zorder>
   <zorder>weatherDescLabel</zorder>
   <zorder>weatherHiLoLabel</zorder>
   <zorder>weatherDownloadButton</zorder>
   <zorder>weatherIconLabel</zorder>
   <zorder>hourLabel</zorder>
   <zorder>secondLabel</zorder>
   <zorder>minuteLabel</zorder>
   <zorder>timeColon</zorder>
   <zorder>degreeLabel</zorder>
   <zorder>currentDateLabel</zorder>
   <zorder>russiaHourLabel</zorder>
   <zorder>russiaTimeColon</zorder>
   <zorder>russiaMinuteLabel</zorder>
   <zorder>israelHourLabel</zorder>
   <zorder>israelTimeColon</zorder>
   <zorder>israelMinuteLabel</zorder>
   <zorder>russiaTimeLabel</zorder>
   <zorder>israelTimeLabel</zorder>
   <zorder>newYorkHourLabel</zorder>
   <zorder>newYorkTimeColon</zorder>
   <zorder>newYorkMinuteLabel</zorder>
   <zorder>newYorkTimeLabel</zorder>
   <zorder>greetingsLabel</zorder>
   <zorder>zipLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2083</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
